test/functional/schema-builder/change-column.ts

```diff
@@ -22,6 +41,49 @@ describe("schema builder > change column", () => {
     beforeEach(() => reloadTestingDatabases(connections))
     after(() => closeTestingConnections(connections))
 
+    it("emits ALTER, not DROP/ADD when increasing varchar length", async () => {
+        for (const base of connections) {
+            // We choose postgres
+            if (base.options.type !== "postgres") continue
```

Comment:
The test needs to run on all the drivers where changing rather than dropping/adding the column is supported.